### DISCUSSION TITLE
New DSA repository

### DISCUSSION THREAD

**Discussion by ming-tsai** [MakeContributions/DSA#2] [General]
This will talk about our new DSA repository of **MakeContributions**, it is related to issue #118

@akrish4 and @paulsonjpaul please add comments or suggestions here.

**Comment by akrish4** [comment]
@ming-tsai  I have created a DSA repository with 2 topics , we have to improve the readme file , shall i transfer it now ?

**Comment by ghost** [comment]
This is a great idea. Currently, Make Contributions only have this repo. I support this discussion

**Comment by akrish4** [comment]
I am getting this error when tried to transfer the repo.
`You don’t have the permission to create public repositories on MakeContributions`
@ming-tsai @paulsonjpaul

**Comment by ghost** [comment]
Maybe it's because you are not a member of this organization

**Reply by akrish4** [reply]
okay , now what to do ? @ming-tsai @paulsonjpaul

**Reply by ghost** [reply]
Let's wait for @ming-tsai. He is the admin.

**Comment by ming-tsai** [comment]
Hi @akrish4 and @paulsonjpaul, sorry about it the github doesn't send me notification, please let me see the path

**Reply by akrish4** [reply]
@ming-tsai  which path ?

**Reply by ming-tsai** [reply]
I was asking for the path of repository, but I found it.

**Comment by ming-tsai** [comment]
This a path of the new [repository](https://github.com/MakeContributions/DSA)
